ods_ci/tests/Tests/400__ods_dashboard/430__data_science_pipelines/434__data-science-pipelines-ui.robot

```diff
@@ -134,7 +136,28 @@ Delete Pipeline Run
     Navigate To Page    Data Science Pipelines    Runs
     Wait Until Page Contains Element    xpath://span[text()='Triggered']
     Click Element    //span[text()='Triggered']
-    Pipelines.Click Action From Actions Menu    ${pipeline_name}    Delete
+    Pipelines.Click Action From Actions Menu    ${run_name}    Delete
     Handle Deletion Confirmation Modal    ${run_name}    triggered run
     Wait Until Page Contains Element    xpath://h2[contains(text(), 'No triggered runs yet')]
-    Capture Page Screenshot
+
+Delete Pipeline
```

Comment:
I think this keyword could be in ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/Pipelines.resource

Reply:
this should to `ODHDashboard/ODHDataSciencePipelines.resource` because this code is not using DS Project UI

Reply:
Done